src/pymovements/gaze/experiment.py

```diff
@@ -109,12 +120,24 @@ def __init__(
             origin=origin,
         )
 
-        checks.check_is_not_none(sampling_rate=sampling_rate)
-        assert sampling_rate is not None
+        checks.check_is_mutual_exclusive(sampling_rate=sampling_rate, eyetracker=eyetracker)
+
+        self.eyetracker = eyetracker
+
+        self._sampling_rate = sampling_rate
+
+        checks.check_is_not_none(sampling_rate=self.sampling_rate)
+        assert self.sampling_rate is not None
 
-        checks.check_is_greater_than_zero(sampling_rate=sampling_rate)
+        checks.check_is_greater_than_zero(sampling_rate=self.sampling_rate)
 
-        self.sampling_rate = sampling_rate
+    @property
+    def sampling_rate(self) -> float:
+        """Assign sampling rate to experiment."""
```

Comment:
This is just a read only property (we don't need a setting method here but it would need the decorator `@sampling_rate.setter` and the value as the parameter). Change the docstring to `Get sampling rate of experiment.`